.gitignore

```
@@ -12,6 +12,9 @@
 
 # external packages
 external/autoware_auto_msgs
+external/autoware_common/
+external/autoware_msgs/
+external/color_names/
 external/tier4_msgs
 external/zmqpp_vendor
```

I think this is simpler and less maintenance 
```suggestion
external/*
!external/lanelet2_matching
!external/concealer
```

---

Fixed in https://github.com/tier4/scenario_simulator_v2/pull/951/commits/71e022ca98c3bea73375a8c5eea10d73db1a61cb

---

OK!